components/feature/addons/src/main/res/values-trs/strings.xml

```diff
@@ -1,17 +1,137 @@
     <string name="mozac_feature_addons_installed_section">Ngà gatûj man</string>
+    <!-- This is displayed in a page where the user can see the installed and recommended(not installed yet) add-ons, this string indicates the recommended section. -->
+    <string name="mozac_feature_addons_recommended_section">Sā sà\'a huin ânj</string>
+    <!-- This is displayed in a page where the user can see the installed and recommended(not installed yet) add-ons, this string indicates the not yet supported section. -->
+    <string name="mozac_feature_addons_unsupported_section">Nitāj si aran\' dugui\'ij ngà nej sa huāa</string>
+    <!-- This is displayed in a page where the user can see the installed and recommended(not installed yet) add-ons, this string indicates the not yet available section. -->
+    <string name="mozac_feature_addons_unavailable_section">Huā nïn\' nitāj si huaj</string>
+    <!-- This is displayed in a page where the user can see the installed and recommended(not installed yet) add-ons, this string indicates the disabled section. -->
+    <string name="mozac_feature_addons_disabled_section">Nitāj si \'iaj sunj</string>
+    <!-- Displays a page with all the details of an add-on. -->
+    <string name="mozac_feature_addons_details">Doj sa huāa</string>
+    <!-- Displays a page with all the permissions of an add-on. -->
+    <string name="mozac_feature_addons_permissions">Sa gāchinj nì\'iô\'</string>
+    <!-- This is button that remove (uninstall an add-on). -->
+    <string name="mozac_feature_addons_remove">Nādure\'</string>
+    <!-- This is the title of a dialog that asks the users if they to install or not an add-on. %1$s is the add-on name. -->
+    <string name="mozac_feature_addons_permissions_dialog_title">¿Nutàt %1$s aj?</string>
+    <!-- This is the subtitle of a dialog that asks the users if they want to install or not an add-on, the subtitle lists the permissions that this add-on requires, the permissions will dynamically be listed after the ":". -->
+    <string name="mozac_feature_addons_permissions_dialog_subtitle">Gīni’ñan dàj gātājt guendâ:</string>
+    <!-- This is a button to add (install) an add-on . -->
+    <string name="mozac_feature_addons_permissions_dialog_add">Nūtà\'</string>
     <!-- This is a button to cancel the add-on installation . -->
     <string name="mozac_feature_addons_permissions_dialog_cancel">Duyichin\'</string>
+    <!-- Accessibility content description to install add-on button. -->
+    <string name="mozac_feature_addons_install_addon_content_description">Gā\'nïnj sa nūtâ\'t riñaj</string>
+    <!-- This is the label of a button to cancel an ongoing add-on installation. -->
+    <string name="mozac_feature_addons_install_addon_dialog_cancel">Dūyichin\'</string>
+    <!-- Indicates how many users have rated an add-on. %1$s will be replaced with number of users -->
+    <string name="mozac_feature_addons_user_rating_count">Nej sû\' arâj sun</string>
```

Comment:
@Pike this string is expected to have a parameter 

```
Errors found:
[task 2020-07-02T00:33:54.571Z]   
[task 2020-07-02T00:33:54.571Z]   /builds/worker/checkouts/src/components/feature/addons/src/main/res/values/strings.xml:109: Error: Inconsistent number of arguments in formatting string mozac_feature_addons_user_rating_count; found both 0 and 1 [StringFormatCount]
[task 2020-07-02T00:33:54.571Z]       <string name="mozac_feature_addons_user_rating_count">Users: %1$s</string>
[task 2020-07-02T00:33:54.571Z]       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[task 2020-07-02T00:33:54.571Z]       /builds/worker/checkouts/src/components/feature/addons/src/main/res/values-trs/strings.xml:106: Conflicting number of arguments here
[task 2020-07-02T00:33:54.571Z]   
[task 2020-07-02T00:33:54.571Z] 
```

Reply:
@Delphine is back, please tag her on issues like this now.

Reply:
@Pike thanks!

Reply:
I've fixed this the best I could in order to unblock this. Waiting for localizer to circle back in case he needs to place the parameter elsewhere in this string.
Anyways, should be fixed for you at next Pontoon sync :) Thanks!